e2e/api/api_test.go

```diff
@@ -149,16 +145,16 @@ func (s *APITestSuite) TestCellsRemovedAfterSwitchAccount() {
 	require.NoError(err)
 
 	for i := 0; i < itersCount; i++ {
-		_, err := s.api.JailManager().NewCell(getChatId(i))
-		require.NoError(err)
+		_, e := s.api.JailManager().NewCell(getChatId(i))
+		require.NoError(e)
```

Comment:
Why `e`, not `err`? :) The last is common.

Reply:
This code contains shadowing variable.

Reply:
And the same reason for other places with this change?

Reply:
Yes, same reason.